runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -187,4 +259,80 @@ private void restoreOriginalDots()
 
 		System.arraycopy(originalDotSprites, 0, mapDots, 0, mapDots.length);
 	}
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.MINIMAP_GROUP_ID)
+		{
+			if (config.leftAlignOrbs() != AlignmentMode.OFF)
+			{
+				if (config.leftAlignOrbs() == AlignmentMode.BOTH)
```

Comment:
All of the if statements with the same body should be moved into one check.

Reply:
Also, maybe this should be a method since it seems like the same code is written multiple times throughout this plugin.